lib/parsers/package-lock-parser.ts

```diff
@@ -75,6 +75,7 @@ export class PackageLockParser implements LockfileParser {
       dependencies: {},
       hasDevDependencies: !_.isEmpty(manifestFile.devDependencies),
       name: manifestFile.name,
+      size: 1,
```

Comment:
This doesn't have to be true because of https://github.com/snyk/nodejs-lockfile-parser/pull/33/files#diff-e2f9058bd2d55f7bf1abe78471eec15aR83

Reply:
It's exactly why it's here. We always have root dependency. So, minimum value is 1.